src/platform.ts

```diff
@@ -153,10 +153,27 @@ export class App {
   async copyTemplate() {
     await promisifyHooks(this.opts.beforeCopy, this.hookArgsWithOriginalResourcesAppDir);
 
-    await fs.copy(this.opts.dir, this.originalResourcesAppDir, {
-      filter: userPathFilter(this.opts),
-      dereference: this.opts.derefSymlinks,
-    });
+    if (this.opts.tmpdir === false) {
```

Comment:
Hey @ianho, we discussed this in an Ecosystem WG meeting and I think this PR is the best solution for https://github.com/electron/forge/issues/3475 as well since it resolves the problem upstream.

Two blockers here:

* Can we add a comment explaining why this is necessary about line 156?
* We should add a test for this code so it doesn't regress in the future.